DataLoader/dev/main.js

```diff
@@ -0,0 +1,19 @@
+import { DataLoader } from '@plantquest/data-loader';
+
+const map = L.map('map').setView([51.505, -0.09], 13);
+const dataLoader = new DataLoader({
+  query: {
+    project_id: 'project_id',
+    plant_id: 'plant_id',
+    stage: 'stage'
+  },
+  senecaPostFunction: SenecaPostFunction, // TODO: SENECA INTEGGRATION
```

Comment:
pass in the entire seneca object

Reply:
Done! Installed seneca lib locally for testing...